app/models/amenity.rb

```diff
@@ -0,0 +1,3 @@
+class Amenity < ActiveRecord::Base
+  belongs_to :room
```

Comment:
Yah I'm pretty sure `Amenity` is going to `have_many` rooms and `Room` is going to `have_many` amenities.

Gonna need a join table.